### PR TITLE
Goalie MergeSCurve

### PR DESCRIPTION
**Pull request by thebhatman**
Goalie on MergeSCurve added. Need to improve it so that it can intercept the moving ball and guard the goal.

### CODE REVIEW

File: belief_state/src/node.cpp [hunk 1]
```diff
@@ -469,7 +469,7 @@ int main(int argc, char **argv)
 {
   // if no argument is passed, assumed our team is blue
   // else if argument 0 = our team blue, 1 = our team yellow
-  ros::init(argc, argv, "beliefstate_node");
+  ros::init(argc, argv, "beliefstate_node1");
```
Comment (saurabhkgp21) [inline belief_state/src/node.cpp]: @thebhatman Why this change?

**Comment by saurabhkgp21** [comment]
@thebhatman  U had used old `node_class.cpp` for changing default position of bots and hence you had undone changes made by shamin to remove unwanted variables in beliefstate.
Just pull updated node_class.cpp and do changes for updating default positions.

**Review by saurabhkgp21 — CHANGES_REQUESTED** [review]

**Comment by thebhatman** [comment]
> @thebhatman U had used old `node_class.cpp` for changing default position of bots and hence you had undone changes made by shamin to remove unwanted variables in beliefstate.
> Just pull updated node_class.cpp and do changes for updating default positions.

Oh Okay! My bad. Sure! Will do that.

**Comment by thebhatman** [comment]
Belief state unwanted variables removed. Node name back to usual. Default bot positions updated so that they do not interfere with playing half field.

**Comment by saurabhkgp21** [comment]
@thebhatman no changes are showing in beliefstate now

**Review by saurabhkgp21 — APPROVED** [review]

**Pull request merged**